extensions/cl_img_memory_management.asciidoc

```diff
@@ -0,0 +1,247 @@
+:data-uri:
+:icons: font
+include::../config/attribs.txt[]
+:source-highlighter: coderay
+
+= cl_img_memory_management
+
+== Name Strings
+
+`cl_img_memory_management`
+
+== Contact
+
+Imagination Technologies Developer Forum: +
+https://forums.imgtec.com/
+
+Tomasz Platek, Imagination Technologies (Tomasz.Platek 'at' imgtec.com)
+
+== Contributors
+
+CY Cheng, Imagination Technologies. +
+Tomasz Platek, Imagination Technologies.
+
+== Notice
+
+Copyright (c) 2024 Imagination Technologies Ltd. All Rights Reserved.
+
+== Status
+
+Draft spec, NOT APPROVED!!
```

Comment:
This should probably be "Final Draft" or "Complete" or "Shipping" or similar.

```suggestion
Complete
```

Reply:
I overlooked it, this should be "Final Draft", already fixed.